wren-ui/src/apollo/server/resolvers/projectResolver.ts

```diff
@@ -466,33 +466,132 @@ export class ProjectResolver {
     const modelColumns =
       await ctx.modelColumnRepository.findColumnsByModelIds(modelIds);
 
-    // Mapping with affected models and columns data into schame change
-    const mappingAffectedToSchemaChange = (changes: DataSourceSchema[]) => {
+    const modelRelationships = await ctx.relationRepository.findRelationInfoBy({
```

Comment:
would it be better if we find relationships by model ids instead of column ids ? I'm thinking, chances are, column ids going to be huge array.